docs/Compiler-Hardening-Guides/Compiler-Options-Hardening-Guide-for-C-and-C++.md

```diff
@@ -394,6 +396,52 @@ Some tools, such as `autoconf`, automatically determine what the compiler suppor
 
 ---
 
+### Warn on Unicode Unicode bidirectional (bidi) override characters |
+
+| [`-Wbidi-chars=any`](#-Wbidi-chars) | GCC | Warn on Unicode bidirectional (bidi) override characters |
+
+| Compiler Flag                                                                              | Supported since            | Description                                                                                  |
+| ------------------------------------------------------------------------------------------ | ----------------------- | -------------------------------------------------------------------------------------------- |
+| <span id="--Wbidi-chars">`-Wbidi-chars=any`</span>                                | GCC | Fortify sources with compile- and run-time checks for unsafe libc usage and buffer overflows         |
```

Comment:
The GCC description of this option also covers variants that warn about bidi characters expressed via universal characters names? Would it make sense to recommend `-Wbidi-char=any,ucn` as that seems to be the most restrictive variant, and include the other variants, including the `unpaired` referred to from the synopsis in this table too? 

Some additional nits:
- The span id is incorrect 
- The correct description seems to be in the misplaced table row above 
- GCC version missing (`-Wbidi-chars` was added in [GCC 12](https://developers.redhat.com/articles/2022/04/25/new-c-features-gcc-12))

Reply:
Proposed changes addressing these suggestion in f62e99ec1b64bde0c92f873a167b9123e09cca66

Reply:
Okay. There's a merge conflict. I'll fix that up.